tests/test_dask_image/test_skexposure/test_histogram_matching.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import absolute_import
+
+import dask.array as da
+import numpy as np
+
+from dask_image.skexposure import histogram_matching
+from dask_image import skexposure
+from skimage import data
+
+from skimage._shared.testing import assert_array_almost_equal, \
+    assert_almost_equal
+
+import pytest
+
+
+@pytest.mark.parametrize('array, template, expected_array', [
+    (da.arange(10, dtype=np.int16, chunks=5),
```

Comment:
It would be good to expand this pytest parameterize to include tests for unsigned integer inputs, and for input images with mismatched (but still valid) datatypes.

Reply:
Good catch!

The unsigned integer test case fails.  I'm commenting it for now, but will debug and fix the code before submitting.